src/main/kotlin/frc/robot/subsystems/gripper/GripperIOSim.kt

```diff
@@ -32,6 +35,13 @@ class GripperIOSim : GripperIO {
         inputs.sensorDistance.mut_replace(
             Units.Centimeters.of(sensorDistance.get())
         )
+
+        if (inputs.appliedVoltage < Units.Volts.zero()) {
+            sensorDistance.set(
+                DEFAULT_SENSOR_SIMULATION_DISTANCE.`in`(Units.Centimeters)
+            )
+        }
```

Comment:
This code is not obvious. Is that just making the sensor return true once the rollers rotate?
It's better to have a trigger that'll handle it. Also, add a small debounce (the coral isn't intake'd instantly).